data_from_players/get_player_data.py

```diff
@@ -0,0 +1,27 @@
+from db.models import Race, Guild
+
+
+def get_player_data(players: dict) -> list:

```

Comment:
why do you need this?

Reply:
I've combined get_player_data function with create_player function.